### DISCUSSION TITLE
How to playback on vlc BBC from this site?

### DISCUSSION THREAD

**Discussion by Bluebuster116** [streamlink/streamlink#4818] [Q&A] [answered]
I am not sure if someone could help me on how to play on VLC the feed from BBC one from this site

http://ds.freestreams-live1.com/bbc-one/

I tried to use the synthax using hls but it doesn't work. Can someone help me please?.

**Comment by bastimeyer** [comment, marked as answer]
https://github.com/streamlink/streamlink/blob/master/CONTRIBUTING.md

> Sites which are hosting stolen content as their main source of content
